src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 4 failures in `PowerShell-CI-linux`
Tests for (error, warning, etc) action preference.Setting ErrorActionPreference to stop prevents user from getting the error exception.$err.Count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 79
79:         It '$err.Count' { $err.Count | Should -Be 1 }
```
Out-Default Tests.'Out-Default -Transcript' shows up in transcript, but not host
```powershell
Expected exactly 'bye', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Core/Out-Default.Tests.ps1: line 19
19:         & $powershell -noprofile -command $script | Should -BeExactly 'bye'
```
Out-Default Tests.Out-Default reverts transcription state when used more than once in a pipeline
```powershell
Expected exactly 'Hello', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Core/Out-Default.Tests.ps1: line 24
24:         & $powershell -noprofile -command "Out-Default -Transcript | Out-Default -Transcript; 'Hello'" | Should -BeExactly "Hello"
```
Out-Default Tests.Out-Default reverts transcription state when exception occurs in pipeline
```powershell
Expected exactly 'Hello', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Core/Out-Default.Tests.ps1: line 28
28:         & $powershell -noprofile -command "try { & { throw } | Out-Default -Transcript } catch {}; 'Hello'" | Should -BeExactly "Hello"
```